tests/integration/nm/ovs_test.py

```diff
@@ -0,0 +1,185 @@
+#
+# Copyright 2019 Red Hat, Inc.
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 2 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <https://www.gnu.org/licenses/>.
+#
+
+from contextlib import contextmanager
+
+import pytest
+
+from libnmstate import nm
+from libnmstate.schema import OvsBridge as OB
+
+from .testlib import mainloop
+
+
+BRIDGE0 = 'brtest0'
+ETH1 = 'eth1'
+
+
+@pytest.fixture
+def bridge_minimum_config():
+    return {
+        OB.CONFIG_SUBTREE: {
+            OB.PORT_SUBTREE: []
```

Comment:
Is an empty port subtree required for the minimum config?

Reply:
This is testing the nm ovs provider module, not nmstate api. For this test, I think we are ok, it just helps handling the ports in the tests uniformly.

The minimum here is in terms of the properties.